tests/phpt/Env.phpt

```diff
@@ -15,6 +15,9 @@ var_dump($env->get('BAR'));
 var_dump($env->get('TEST'));
 
 --EXPECT--
+Standard input code:7:
```

Comment:
I guess this is what you meant when talking about the extra PHPUnit output. I tried to figure out what causes that, but in the end I said to myself that it doesn't really matter anyway.